src/states/adult/mod.rs

```diff
@@ -387,6 +387,16 @@ impl Base for Adult {
         &mut self.rng
     }
 
+    fn finish_handle_input(&mut self, outbox: &mut dyn EventBox) -> Transition {
```

Comment:
Yes, I reckon these states will go eventually as here perhaps we should return an error. Happy for that later though.